smrf/utils/utils.py

```diff
@@ -448,3 +447,36 @@ def grid_interpolate_deconstructed(tri, values, grid_points, method='linear'):
         return CloughTocher2DInterpolator(tri, values)(grid_points)
     elif method == 'linear':
         return LinearNDInterpolator(tri, values)(grid_points)
+
+
+def apply_utm(s, force_zone_number):
+    """
+    Calculate the utm from lat/lon for a series
+
+    Args:
+        s: pandas series with fields latitude and longitude
+        force_zone_number: default None, zone number to force to
+
+    Returns:
+        s: pandas series with fields 'X' and 'Y' filled
+    """
+    p = utm.from_latlon(s.latitude, s.longitude,
+                        force_zone_number=force_zone_number)
+    s['utm_x'] = p[0]
+    s['utm_y'] = p[1]
+    return s
+
+
+def date_range(start_date, end_date, increment):
+    """
+    Calculate a list between start and end date with
+    an increment
+    """
+    result = []
+    nxt = start_date
+
+    while nxt <= end_date:
+        result.append(nxt)
+        nxt += increment
+
+    return np.array(result)
```

Comment:
I feel that I just recently removed this method in favor of 
```python
list(pd.date_range(
            self.start_date,
            self.end_date,
            freq="{[time][time_step]}min".format(self.config),
            tz=self.time_zone
        ))
```

Reply:
Haha, you did. Moved your function to utils to allow it to be reused.